src/radical/pilot/agent/scheduler/base.py

```diff
@@ -121,9 +121,15 @@
 #       'cpu_threads'      : 2,
 #       'gpu_processes     : 2,
 #       'slots' :
-#       {                 # [[nodename, [node_uid], [core indexes],   [gpu idx]]]
-#         'nodes'         : [[node_1,   node_uid_1, [[0, 2], [4, 6]], [[0]    ]],
-#                            [node_2,   node_uid_2, [[1, 3], [5, 7]], [[0]    ]]],
+#       {                 # [[nodename, [node_uid], 
+                          # [[core indexes for process 1],.., [core indexes for process 4]],   
+                          # [[gpu idx for process 1], .., [gpu idx for process 4]]]
+#         'nodes'         : [[node_1,   node_uid_1, 
+                          # [[0, 2], [4, 6]], 
+                          # [[0]]],
+#                           [node_2,   node_uid_2, 
+                          # [[1, 3], [5, 7]], 
+                          # [[0]    ]]],
```

Comment:
Uh, is this really more readable?

Reply:
I iterated on it once more to match the lines. I think it is more legible, but this is subjective. I'll let you make the final call :)